docs/core/diagnostics/dumps.md

```diff
@@ -13,7 +13,7 @@ A dump is a file that contains a snapshot of the process at the time it was crea
 Dumps can be collected in a variety of ways depending on which platform you are running your app on.
 
 > [!NOTE]
-> Collecting a dump inside a container requires PTRACE capability, which can be added via `--cap-add=SYS_PTRACE` or `--privileged`.
+> Collecting a dump requires that the process have access to ptrace targeting parent processes. This means /proc/sys/kernel/yama should have ptrace_scope set to 1 or 0. If running under any type of Open Container Initiatory technology, the seccomp profile must allow for calls to ptrace. PTRACE capability can be added via `--cap-add=SYS_PTRACE` or `--privileged`. Afterwards, the dump save path config should point out that the user starting the process needs to have write access to the directory configured as the dump path.
```

Comment:
I'd move the path to another point here. This note is purely a point on ptrace, and the 4.8 example is still pertinent I feel. Regarding the path one, maybe the notes on the environment variable are a better place for it. These caveats need to be visible somehow from the different dump docs (i.e dotnet-dump also needs similar notices).

Reply:
I don't know what the 4.8 example is referring to. 
What do you think about creating a file called NotesAboutDumps (or something else) and have every dump page have a link to that file?

Reply:
Having a dump FAQ seems sensible to me. As for the 4.8, it refers to the kernel version running the application (docker and other container solutions block ptrace on 4.8 and below as they had security layering issues prior to that and the pressure valve they provided for people that needed it was to require CAP_SYS_PTRACE).